### ISSUE TITLE
How to add images inside articles?

### ISSUE THREAD

**Issue by nurzhannogerbek** [bluejamesbond/TextJustify-Android#71]
Hello! First of all Thank you for this project!

Is there any way to add images inside articles? As I see everything was made programmally without xml. What can you advice?

**Comment by sigrlami** [comment]
@NogerbekNurzhan
No, but you can always create layouts programmatically, put everything you need in `ScrollView` then add `DocumentView`, `ImageView`, `DocumentView`...
